Feature base layout for website.

pniewiarowski opened the pull request:
# Layout
Declare base layout with footer and navbar, add two atoms for typography and navbar, hero and footer organisms.

pniewiarowski committed:
Setup the directory structure based on theNuxt documentation guide.

pniewiarowski committed:
Replace App.vue with pages/index.vue to support Muxt pages routing.

pniewiarowski committed:
Install and setup nuxt modules for UI, i18n, icons, images and fonts.

pniewiarowski committed:
Fix formatting for config file.

pniewiarowski committed:
Install nuxt/icon as dependency for project.

pniewiarowski committed:
Declare base layout including navbar and footer.

mateuszhorczak commented on components/organisms/footer.vue:
Navbar w `footer.vue`?

mateuszhorczak commented on layouts/default.vue:
```diff
@@ -0,0 +1,15 @@
+<template>
+  <UApp>
+    <Navbar/>
+    <slot/>
```
<template>
  <UApp>
    <Navbar/>
        <NuxtPage />
    <Footer/>
  </UApp>
</template>

mateuszhorczak committed:
Update default.vue

w layout.vue wrzucasz NuxtPage tam gdzie chcesz by sie strona z pages dir renderowała

mateuszhorczak reviewed (changes requested):
change navbar string in footer to navbar

pniewiarowski committed:
Change heading slut value in footer organism.

pniewiarowski commented:
Fixed.
